test/project_types/script/commands/enable_test.rb

```diff
@@ -12,8 +12,8 @@ def setup
         @configuration = { entries: [] }
         @ep_type = 'discount'
         @script_name = 'script'
-        @api_key = 'key'
-        @shop_domain = 'shop.myshopify.com'
+        @api_key = 'apikey'
+        @shop_domain = 'my-test-shop.myshopify.com'
```

Comment:
I think we also should stub out the `env_file` entries like you have done in `push_test`.